libs/cli/langchain_cli/namespaces/template.py

```diff
@@ -122,14 +121,12 @@ def serve(
         else "langchain_cli.dev_scripts:create_demo_server_configurable"
     )
 
-    command = [
-        "uvicorn",
-        "--factory",
+    import uvicorn
+
+    uvicorn.run(
         script,
-        "--reload",
-        "--port",
-        port_str,
-        "--host",
-        host_str,
-    ]
-    subprocess.run(command)
+        factory=True,
+        reload=True,
+        port=port if port is not None else 8000,
```

Comment:
should we change the default port away from 8000?

Reply:
let's discuss. Gonna release this to help with going through all the current packages, and then we can change default port in a future release.